db/context.go

```diff
@@ -53,6 +53,11 @@ func ensureContextTxn(ctx context.Context, db transactionDB, readOnly bool) (con
 	return SetContextTxn(ctx, txn), txn, nil
 }
 
+// mustGetContextTxn returns the transaction from the context or panics.
+func mustGetContextTxn(ctx context.Context) datastore.Txn {
```

Comment:
suggestion: This feels like an odd place for such a function, why is this defined privately in `db`, instead of internal-public in `core`?  Is the long term plan to move it, or to duplicate it? Or have we managed to remove all `txn` usage from all other packages?

Reply:
From the standup discussion, our consensus was to use this only within the `db` package. Other APIs will still use `txn` params. I agree that it should move to `core` if we decide to expand it to other areas.